src/frontend/staff-portal/src/app/components/staff-workbench/ticket-info/ticket-info.component.ts

```diff
@@ -240,7 +244,7 @@ export class TicketInfoComponent implements OnInit {
   // return a filtered list of statutes
   public filterStatutes(val: string): Statute[] {
     if (!this.lookupsService.statutes || this.lookupsService.statutes.length == 0) return [];
-    return this.lookupsService.statutes?.filter(option => option.__statuteString.indexOf(val) >= 0);
+    return this.lookupsService.statutes?.filter(option => (option.__statuteString || "").toUpperCase().indexOf((val || "").toUpperCase()) >= 0);
```

Comment:
`return this.lookupsService.statutes.filter(option => (option.__statuteString || "").toUpperCase().indexOf((val || "").toUpperCase()) >= 0);`

and `option.__statuteString` can be null ?